client/src/components/Card.js

```diff
@@ -26,10 +28,6 @@ export default function VaccinationCard({
     );
   }
 
-  function handleClick(clickedVacc) {
-    onSetEdited(clickedVacc);
-  }
-
   return (
```

Comment:
Please check the date component. It shows in deploment the wrong date when it is recalled.